stregsystem/stregsystem/views.py

```diff
@@ -0,0 +1,28 @@
+from django.http import HttpResponse
+from django.template import RequestContext
+from django.template.loader import get_template
+from catalog.models import *
+
+def index(request):
+    template = get_template('stregsystem/index.html')
+    listings = Listing.objects.all()
+    variables = RequestContext(request, {
+        'head_title': 'Stregsystem',
+        'products': listings,
+    })
+    output = template.render(variables)
+    return HttpResponse(output)
```

Comment:
Some activity, wooho :)

This way of rendering views is a bit old and not really used anymore. A rewritten index function with a modern approach (though still function based) could be something along the lines of:

```python
from django.shortcuts import render

def index(request):
    args = {
        'head_title': 'Stregsystem',
        'products': Listing.objects.all()
    }
    return render(request, 'stregsystem/index.html', args)
```

You might want to consider not sending the head title with all responses, though. I'll comment below on a prettier way to handle this :)